internal/csi-addons/rbd/volumegroup.go

```diff
@@ -527,6 +597,21 @@ func (vs *VolumeGroupServer) ModifyVolumeGroupMembership(
 		}
 	}
 
+	// Enable mirroring after modification of volume group
+	if isMirroringEnabled {
```

Comment:
`isMirroringEnabled` is a local variable

what if there is a restart during a modifymembership call with change in PVCs in group ? 

Is that scenario tested ? 
Does it go smoothly ?

Reply:
This is not global variable its per RPC call, Do you see any problem with it?

Reply:
> This is not global variable its per RPC call, Do you see any problem with it?

My questions are mentioned in the first comment.

The expectation form the variable is that if mirroring was enabled before modifymembership call, mirroring will be switched back on at the end of modification.
If there is a restart, when the new modify call begins, mirroring will not be enabled, thus mirroring will not get switched back on at the end.

Reply:
Yes, This is for dynamic grouping, CSI will first disable mirroring add images to the group and reenable the mirroring. If the mirroring is already disabled it will just add the images to the group and enable the mirroring again.

Reply:
got it, Thanks for catching this. i miss read it.

Reply:
Updated the code to remove the check from enable mirroring completely.
We should not bother checking on enable mirroring, as it will return success even if it's enabled.
So, we should be good with this.